modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CSharpClientCodegen.java

```diff
@@ -127,6 +116,10 @@ public CSharpClientCodegen() {
         addSwitch(CodegenConstants.OPTIONAL_EMIT_DEFAULT_VALUES,
                 CodegenConstants.OPTIONAL_EMIT_DEFAULT_VALUES_DESC,
                 this.optionalEmitDefaultValue);
+
+        addSwitch(CodegenConstants.GENERATE_PROPERTY_CHANGED,
+                CodegenConstants.PACKAGE_DESCRIPTION_DESC,
```

Comment:
@jimschubert I believe `PACKAGE_DESCRIPTION_DESC` should be replaced by `GENERATE_PROPERTY_CHANGED_DESC` instead.